### ISSUE TITLE
Is this a wrong sentence of "self.mode == 'linear'" 

### ISSUE THREAD

**Issue by gll89** [dmlc/gluon-cv#1374]
In the script of "utils/lr_scheduler.py ", the Line 129 displays "self.mode =='linear'", but in parse, lr_model is just one of 'step', 'poly' or 'cosine'. Is there something wrong in the Line 129? Thanks!

**Comment by gll89** [comment]
The mode of 'linear' means learning rate warmup?
